js/my.js

```diff
@@ -72,9 +72,24 @@ document.addEventListener("DOMContentLoaded", function () {
   if (localStorage.host) {
     $("host").value = localStorage.host;
   }
-    if (localStorage.channels) {
+  if (localStorage.channels) {
     $("channels").value = localStorage.channels;
   }
+  if (localStorage.port) {
+    $("port").value = localStorage.port;
+  }
+
+  /* Local Storage is strings! */
+  if (!!+localStorage.secure) {
```

Comment:
the cast to number is not necessary; it was placed into localstorage as a number on L143.  The cast to boolean is not necessary.  Let's remove the comment and cast.